### PR TITLE
Ecu+sleep

### PR DESCRIPTION
**Pull request by MBcode**
this includes setting S3ADDRESS for a different bucket location for repo2summary.sh run
or from 2nd arg if you are just calling the get_repo.sh part

fnq.py adds new 'sleep' if the repo.nq file it is loading into fuseki is very big
to tsum.py won't go too soon

I have something to do the upload, which needs testing, and I'd rather get in the next PR

### CODE REVIEW

File: summarize/src/get_repo.py [hunk 1]
```diff
@@ -2,16 +2,108 @@
...
 import os
+##import ec
+#import earthcube_utilities as ec  #make sure this is availabe
+##import ../earthcube_utilities as ec  #assuming it is one level above
 #for now keeping in a dir named after each repo, so next step can run2nq over it
  #where it takes the it's filename, to turn the triples into quads w/that name
+#==from ec.py in case I don't want to import it:
+cwd = os.getcwd()
+ncsa_minio = "https://oss.geocodes.ncsa.illinois.edu/"
+S3ADDRESS=os.getenv("S3ADDRESS")
```
Comment (valentinedwv) [inline summarize/src/get_repo.py:13]: Prefix the keys, say ECU_S3ADDRESS

File: summarize/src/get_repo.py [hunk 2]
```diff
@@ -2,16 +2,108 @@
...
+        else:
+            print(f'have:{fl} already')
+    if dbg: #might dump this all time, or by arg
+        list2txtfile("l1h",files)
+    return files
+
+#=end utils
+
+#def get_repo(repo):
+def get_repo(repo, default_bucket= "https://oss.geocodes.ncsa.illinois.edu/"):
```
Comment (valentinedwv) [inline summarize/src/get_repo.py:99]: bucket or server ECU_S3ADDRESS/?

File: summarize/src/get_repo.py [hunk 3]
```diff
@@ -2,16 +2,108 @@
...
+          key=s3key,
+          secret=s3secret,
+          #client_kwargs = {"endpoint_url":"https://oss.geodex.org"}
+          #client_kwargs = {"endpoint_url":"https://oss.geocodes-dev.earthcube.org/"}
+          client_kwargs = {"endpoint_url": minio_endpoint_url}
+       )
+    return oss
+
+#def oss_ls(path='test3/summoned',full_path=True,minio_endpoint_url="https://oss.geocodes-dev.earthcube.org/"):
+def oss_ls(path='gleaner/milled',full_path=True,minio_endpoint_url="https://oss.geocodes.ncsa.illinois.edu/"):
```
Comment (valentinedwv) [inline summarize/src/get_repo.py:56]: should there be an ability to pass a --path --bucket to the cli

File: summarize/src/get_repo.py [hunk 4]
```diff
@@ -21,15 +113,37 @@ def get_repo(repo):
...
     import sys
+    print(f'argv={sys.argv}')
+    default_bucket= "https://oss.geocodes.ncsa.illinois.edu/"
+    s3=os.getenv("S3ADDRESS") 
+    #if ec.is_str(s3):
+    if is_str(s3):
+        #if not ec.is_http(s3):
+        if not is_http(s3):
+            s3="https://" + url_w_end_slash(s3)
+        print(f'set default_bucket to S3ADDRESS={s3}')
```
Comment (valentinedwv) [inline summarize/src/get_repo.py:137]: This is server... bucket is gleaner, path is milled/{repo}

File: summarize/src/get_repo.py (hunk 1 above)
Comment (MBcode) [reply summarize/src/get_repo.py:13]: added prefix

File: summarize/src/get_repo.py (hunk 2 above)
Comment (MBcode) [reply summarize/src/get_repo.py:99]: just server, right now path set to 'gleaner/milled' then whichever '/repo'
so will just rename var

File: summarize/src/get_repo.py (hunk 3 above)
Comment (MBcode) [reply summarize/src/get_repo.py:57]: for now it comes from 'gleaner/milled' + '/repo'. 
but I can add this in the next PR, once I can add the upload as well

File: summarize/src/get_repo.py (hunk 4 above)
Comment (MBcode) [reply summarize/src/get_repo.py]: resolved from 1st comment about it/tnx!

**Pull request merged**